legend-engine-language-pure-dsl-service/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/ServiceParserGrammar.g4

```diff
@@ -201,4 +202,22 @@ testSingleValueParam:                        primitiveValue
 ;
 
 testParam:                              testListValueParam | testSingleValueParam
+;
+
+
+// -------------------------------------- TELEMETRY --------------------------------------
+
+serviceTelemetry:                       SERVICE_TELEMETRY COLON
+                                            BRACE_OPEN
+                                                (
+                                                    metricGroups
+                                                )?
```

Comment:
* instead of ?, similar to convention in other places, so that we can add new field, if needed later